### DISCUSSION TITLE
Problema riferimenti con Nota di Credito (od_number e od_date)

### DISCUSSION THREAD

**Discussion by Drakas** [fattureincloud/api#192] [Q&A] [answered]
Buonasera,
Ho i seguenti problemi quando genero una Nota di Credito:
1) Non viene valorizzato il campo od_number
2) Ottengo Invalid date al posto della corretta valorizzazione del campo od_date

La Nota viene comunque creata su Fatture in Cloud e ma ottengo il risultato errato come da screen (campo vuoto e invalid date nel secondo campo), allego il payload interessato:

		"e_invoice": true,
		"ei_data": {
			"payment_method": "MP05",
			"bank_iban": "IT585894985984859489",
			"od_number": "264",
			"od_date": "2022-12-15"
		},
		"use_gross_prices": true,
		"show_paypal_button": false,
		"show_tspay_button": false,
		"type": "credit_note"

<img width="281" alt="Immagine 2022-12-15 193028" src="https://user-images.githubusercontent.com/3465823/207940501-ba703b3e-3895-4fbb-bab0-3aa3eb8a9ee2.png">

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
i due campi indicati nella sua immagine corrispondono ai seguenti campi JSON: "invoice_number" e "invoice_date"

Mi faccia sapere se questo risolve il suo problema.

Grazie
Mauro
